src/common/verbose.cpp

```diff
@@ -308,29 +307,8 @@ bool get_verbose_timestamp() {
         verbose_timestamp.set(val);
     }
     return verbose_timestamp.get();
-#endif
 }
 
-#if defined(DISABLE_VERBOSE)
-void pd_info_t::init(
-        dnnl::impl::engine_t *, const dnnl::impl::primitive_desc_t *) {}
-
-std::string rt_mds2str(primitive_kind_t prim_kind, const memory_desc_t *src_md,
-        const memory_desc_t *wei_md, const memory_desc_t *bia_md,
-        const memory_desc_t *dst_md) {
-    return std::string();
-}
-
-std::string rt_dims2fmt_str(primitive_kind_t prim_kind,
-        const memory_desc_t *src_md, const memory_desc_t *wei_md,
-        const memory_desc_t *dst_md) {
-    return std::string();
-}
-
-void verbose_printf_impl(const char *raw_fmt_str, verbose_t::flag_kind kind) {}
```

Comment:
Could you clarify why these were not enough? At a glance, it seems that under DISABLE_VERBOSE, the behavior after the patch should be similar (return empty string or empty function).

Reply:
The number of functions was growing, they are used across different places in verbose-reporting functionality. I decided that instead of handling each function under `if (disabled)` section (including future added) separately, it's just easier to force some of them to return empty strings under that build flag and call it a day. Should be no issues with future functions.